src/main/java/com/ioet/bpm/people/utils/PasswordStorage.java

```diff
@@ -0,0 +1,108 @@
+package com.ioet.bpm.people.utils;
+
+/*
+ * Copyright (C) 2017 Dominik Schadow, dominikschadow@gmail.com
+ *
+ * This file is part of the Java Security project.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.util.Base64;
+
+
+public class PasswordStorage {
+    private static final Logger log = LoggerFactory.getLogger(PasswordStorage.class);
```

Comment:
I am not a big fan of static methods, I bet there is a better way to do this. 
Check this out: https://www.yegor256.com/2014/05/05/oop-alternative-to-utility-classes.html